From dff26dea758535a1f5e0cb3279ddc1d5a6191b56 Mon Sep 17 00:00:00 2001
From: Sahana Prasad <sahana@redhat.com>
Date: Mon, 12 Sep 2022 11:07:38 +0200
Subject: [PATCH 5/5] Apply patch stunnel-5.56-curves-doc-update.patch

Patch-name: stunnel-5.56-curves-doc-update.patch
Patch-id: 6
From-dist-git-commit: 70b3076eb09912b3a11f371b8c523303114fffa3
---
 doc/stunnel.8.in       | 2 ++
 doc/stunnel.html.in    | 2 ++
 doc/stunnel.pl.8.in    | 2 ++
 doc/stunnel.pl.html.in | 2 ++
 doc/stunnel.pl.pod.in  | 2 ++
 doc/stunnel.pod.in     | 2 ++
 6 files changed, 12 insertions(+)

diff --git a/doc/stunnel.8.in b/doc/stunnel.8.in
index e74e174..03b503b 100644
--- a/doc/stunnel.8.in
+++ b/doc/stunnel.8.in
@@ -490,6 +490,8 @@ This file contains multiple CRLs, used with the \fIverifyChain\fR and
 .IX Item "curves = list"
 ECDH curves separated with ':'
 .Sp
+Note: This option is supported for server mode sockets only.
+.Sp
 Only a single curve name is allowed for OpenSSL older than 1.1.1.
 .Sp
 To get a list of supported curves use:
diff --git a/doc/stunnel.html.in b/doc/stunnel.html.in
index df0efdd..385ac8d 100644
--- a/doc/stunnel.html.in
+++ b/doc/stunnel.html.in
@@ -596,6 +596,8 @@
 
 <p>ECDH curves separated with &#39;:&#39;</p>
 
+<p>Note: This option is supported for server mode sockets only.</p>
+
 <p>Only a single curve name is allowed for OpenSSL older than 1.1.1.</p>
 
 <p>To get a list of supported curves use:</p>
diff --git a/doc/stunnel.pl.8.in b/doc/stunnel.pl.8.in
index 4efe602..9683b4c 100644
--- a/doc/stunnel.pl.8.in
+++ b/doc/stunnel.pl.8.in
@@ -494,6 +494,8 @@ przez opcje \fIverifyChain\fR i \fIverifyPeer\fR.
 .IX Item "curves = lista"
 krzywe ECDH odddzielone ':'
 .Sp
+Uwaga: ta opcja wpływa tylko na gniazda w trybie serwera.
+.Sp
 Wersje OpenSSL starsze niż 1.1.1 pozwalają na użycie tylko jednej krzywej.
 .Sp
 Listę dostępnych krzywych można uzyskać poleceniem:
diff --git a/doc/stunnel.pl.html.in b/doc/stunnel.pl.html.in
index 8e40042..3025e9f 100644
--- a/doc/stunnel.pl.html.in
+++ b/doc/stunnel.pl.html.in
@@ -586,6 +586,8 @@
 
 <p>krzywe ECDH odddzielone &#39;:&#39;</p>
 
+<p>Uwaga: ta opcja wpływa tylko na gniazda w trybie serwera.</p>
+
 <p>Wersje OpenSSL starsze ni&#x17C; 1.1.1 pozwalaj&#x105; na u&#x17C;ycie tylko jednej krzywej.</p>
 
 <p>List&#x119; dost&#x119;pnych krzywych mo&#x17C;na uzyska&#x107; poleceniem:</p>
diff --git a/doc/stunnel.pl.pod.in b/doc/stunnel.pl.pod.in
index 4419f9f..c48387a 100644
--- a/doc/stunnel.pl.pod.in
+++ b/doc/stunnel.pl.pod.in
@@ -535,6 +535,8 @@ przez opcje I<verifyChain> i I<verifyPeer>.
 
 krzywe ECDH odddzielone ':'
 
+Uwaga: ta opcja wpływa tylko na gniazda w trybie serwera.
+
 Wersje OpenSSL starsze niż 1.1.1 pozwalają na użycie tylko jednej krzywej.
 
 Listę dostępnych krzywych można uzyskać poleceniem:
diff --git a/doc/stunnel.pod.in b/doc/stunnel.pod.in
index 1a49d42..7a92697 100644
--- a/doc/stunnel.pod.in
+++ b/doc/stunnel.pod.in
@@ -533,6 +533,8 @@ I<verifyPeer> options.
 
 ECDH curves separated with ':'
 
+Note: This option is supported for server mode sockets only.
+
 Only a single curve name is allowed for OpenSSL older than 1.1.1.
 
 To get a list of supported curves use:
-- 
2.46.0

